sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2462,6 +2489,27 @@ object Radar {
             return
         }
 
+        if (RadarSettings.getPermissionsDenied(context)) {
+            sendLogConversionRequest(name, data, callback = callback)
+
+            return
+        }
+
+        val timestampSeconds = System.currentTimeMillis() / 1000
+        val lastTrackedTime = RadarSettings.getLastTrackedTime(context)
+        if (lastTrackedTime - timestampSeconds < 60) {
+            sendLogConversionRequest(
+                name,
+                data,
+                cachedTrackInfo.location,
+                cachedTrackInfo.events,
+                cachedTrackInfo.user,
```

Comment:
I don't believe we need `cachedTrackInfo` at all. Basically, if a track call was in the last minute, the location and other user context (geofence, place, etc) will be enriched onto the event server side

Reply:
we would only need the name and data e.g. pseudo `sendLogConversionRequest('in-app-purchase', { 'amount': 10 })`